core/templates/dev/head/domain/skill/ConceptCardBackendApiService.ts

```diff
@@ -101,16 +101,16 @@ oppia.factory('ConceptCardBackendApiService', [
        * Returns whether the given concept card is stored within the local data
        * cache or if it needs to be retrieved from the backend upon a laod.
        */
-      isCached: function(skillId) {
-        return _isCached(skillId);
+      isCached: function(skillIds) {
```

Comment:
Are you checking if multiple skillIds are cached here?

Reply:
Yes.

Reply:
Why exactly?

Reply:
See my above comment about why we shouldn't be doing this